en/docs/develop/creating-artifacts/creating-an-api.md

```diff
@@ -233,45 +234,64 @@ The service's metadata is used by the API management runtime to generate the API
 
 -->
 
-### Design the integration
+## Design the integration
+
+1. Open the **Resource View** of the API resource.
+
+    1. Go to **MI Project Explorer** > **APIs**.
+    2. Under the API you created, click the API resource to open the **Resource View** of the API resource.
+
+2. Click on the **+** icon below the API resource to open the mediator palette.
+
+3. Add the required mediators and connectors to the API resource and design the integration flow.
+
+## Update properties
+
+### Update the API properties
+
+1. Go to **MI Overview**.
+
+2. Under **APIs**, select the API you want to edit. This will open the **Service Designer**.
+
+3. On the **Service Designer**, click the **Edit** icon to edit the API. 
 
-1. On the **Service Designer**, click on the API resource and go to the **Resource View**.
+    <a href="{{base_path}}/assets/img/develop/create-artifacts/create-rest-api/edit-api-resource-interface.png"><img src="{{base_path}}/assets/img/develop/create-artifacts/create-rest-api/edit-api-resource-interface.png" alt="available-apis" width="80%"></a>
 
-2. Add the required mediators and connectors to the API resource and design the integration flow.
+    This will open the **Edit API** interface.
 
-You can also use the [**Source View**](#using-the-source-view) or the [**Swagger View**](#using-the-swagger-editor) to update the API configuration.
+4. Once you edit, click **Save Changes**.
 
-### Update properties
+    See the complete list of [API properties]({{base_path}}/reference/synapse-properties/rest-api-properties/#rest-api-properties) you can configure.
 
-To update API level properties:
+### Update the API resource properties
```

Comment:
Let's rename properties to configuration

Reply:
Fixed via https://github.com/wso2/docs-mi/pull/1151/commits/4d790beff54453c1054d81f07b4a360bbeada5b4.